src/client/main.ts

```diff
@@ -2,11 +2,14 @@
  * @module Index
```

Comment:
Suggest a pass over the comments in example code in this file, varies between `//Comment`, `// comment`, and `//comment`